optuna_dashboard/static/components/StudyDetail.tsx

```diff
@@ -15,9 +17,10 @@ import {
   IconButton,
   Select,
   MenuItem,
+  FormGroup,
 } from "@material-ui/core"
-import { Home, Cached } from "@material-ui/icons"
-
+import { Home, Cached, Settings } from "@material-ui/icons"
+import CloseIcon from "@material-ui/icons/Close"
```

Comment:
How about importing `CloseIcon` component like L22?

Reply:
It's not included in the corresponding type definition files, so we may have to keep this line.
```
ERROR in /Users/huzi/Workspace/github/optuna-dashboard/optuna_dashboard/static/components/StudyDetail.tsx
./optuna_dashboard/static/components/StudyDetail.tsx 22:33-42
[tsl] ERROR in /Users/huzi/Workspace/github/optuna-dashboard/optuna_dashboard/static/components/StudyDetail.tsx(22,34)
      TS2305: Module '"@material-ui/icons"' has no exported member 'CloseIcon'.
 @ ./optuna_dashboard/static/components/App.tsx 5:0-44 12:119-130
 @ ./optuna_dashboard/static/index.tsx 3:0-39 5:24-27
```

Reply:
I see! It may be an issue of type stub file. Anyway, I merged the pull request.